x-pack/plugins/enterprise_search/common/types/engines.ts

```diff
@@ -51,7 +51,6 @@ export interface SchemaFieldIndex {
 }
 
 export interface SchemaField {
```

Comment:
Can we remove `EnterpriseSearchSchemaField` type now that we've updated the frontend page?

Or it maybe better to remove the current type and rename this `SchemaField` to `EnterpriseSearchSchemaField`

Reply:
✅ https://github.com/elastic/kibana/pull/153602/commits/09be805d68a61ca0cc876879c910b0ad1659792b